specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/examples/IntegrationAccountLargeMaps_CreateOrUpdate.json

```diff
@@ -0,0 +1,66 @@
+{
+  "parameters": {
+    "api-version": "2019-05-01",
+    "resourceGroupName": "testResourceGroup",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "integrationAccountName": "testIntegrationAccount",
+    "mapName": "testMap",
+    "map": {
+      "properties":{
+         "mapType":"Xslt",
+         "contentLink":{
+            "uri":"<Blob SAS URL of the map>"
+         },
+         "contentType":"application/xml",
+         "metadata":{}
+      },
+      "location":"westus"
+   } 
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "mapType": "Xslt",
+          "contentLink": {
+            "uri": "<Uri>",
```

Comment:
URI for what?

```suggestion
            "uri": "<URI>",
```

Reply:
@ecfan 

It shall be "Blob URI with SAS token".  The blob refers to the map blob.